inst/tests/tests.Rraw

```diff
@@ -2186,7 +2186,7 @@ if (ncol(DT)==2L) setnames(DT,c("A","B")) # else don't stop under torture with s
 test(714, DT[,z:=6:10], data.table(A=1:5,B=5,z=6:10))
 
 # Test J alias is now removed outside DT[...] from v1.8.7 (to resolve rJava::J conflict)
-test(715, J(a=1:3,b=4), error="J() called outside of [.data.table. J() is only intended for use in i.")
+test(715, J(a=1:3,b=4), error=base_messages$missing_function("J"))
```

Comment:
```suggestion
test(715, J(a=1:3,b=4), error=base_messages$missing_function("J"))
# future:
# test(715, J(a=1:3,b=4), error="J() called outside of [.data.table. J() is only intended for use in i.")
```

Reply:
this will cause an error because we aren't exporting J
```
Test 715 didn't produce the correct error:
Expected: J() called outside of [.data.table. J() is only intended for use in i.
Observed: could not find function "J"
```
so i added it to list where internal-only non exposed functions are imported 

```
  haszlib = data.table:::haszlib
  J = data.table:::J 
```
is this better or just not changing error ?

Reply:
sorry, I think you misread the suggested edit -- in mine the "new" test is commented out.

Reply:
> sorry, I think you misread the suggested edit -- in mine the "new" test is commented out.

oh i am sorry , i didn't notice 😓

Reply:
done, plz check it out

Reply:
I don't know how this passed, but test 715 is now failing locally on my system and I think it should have failed in our CI too.

Since we leave the `J` function in line 2773 the behavior should already be the future behavior?

Reply:
I don't know what's happening but current master branch doesn't seem to be failing test 715 in my local system, 

```
All 11202 tests (last 2264.8) in tests/tests.Rraw completed ok in 14.6s elapsed (14.2s cpu)
```

Reply:
Hi @ben-schwen are you still facing this issue, plz inform me if I can be of help :)